capture/writer-simple.c

```diff
@@ -380,6 +381,10 @@ LOCAL void writer_simple_write(const MolochSession_t * const session, MolochPack
         if (config.debug)
             LOG("opened %d %s %d", thread, name, info->file->fd);
         g_free(name);
+
+        MOLOCH_LOCK(simpleQ);
```

Comment:
Shouldn't need this LOCK/UNLOCK since writer_simple_check is called inside each packet thread just like writer_simple_check. I guess should add a comment to the top of writer_simple_check

Reply:
Thanks! I wasn't sure. Fixed!